docs/dev/building-on-zksync/hello-world.md

```diff
@@ -124,13 +116,13 @@ contract Greeter {
 }
 ```
 
-5. Compile the contract with the following command:
+3. Compile the contract with the following command:
 
 ```sh
 yarn hardhat compile
 ```
 
-6. Create the following deployment script in `deploy/deploy.ts`:
+4. Create the following deployment script in `deploy/deploy.ts`, replacing the private key placeholder with your own:
 
```

Comment:
```suggestion
4. Create the following deployment script in `deploy/deploy.ts`, replacing the private key placeholder with your own:

```

I think it's nicer to highlight the placeholder text(WALLET-PRIVATE-KEY) in this paragraph in any way possible so the reader doesn't have to assume what it is.